go-controller/vendor/github.com/google/gnostic-models/openapiv3/annotations.proto

```
@@ -0,0 +1,56 @@
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package openapi.v3;
+
+import "google/protobuf/descriptor.proto";
+import "openapiv3/OpenAPIv3.proto";
```

Buf (1.55.1)

import "openapiv3/OpenAPIv3.proto": file does not exist

(COMPILE)